helm-chart/eoapi/templates/services/deployment.yaml

```
@@ -71,6 +71,14 @@ spec:
             name: {{ $secret }}
         {{- end }}
         {{- end }}
+      {{- with index $v "settings" "affinity" }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with index $v "settings" "tolerations" }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Can we check for the existence of `affinity` and `tolerations`, and possibly avoid adding them as empty objects to `values.yaml`?

```suggestion
        {{- if index $v "settings" "affinity" }}
        affinity:
          {{- toYaml (index $v "settings" "affinity") | nindent 8 }}
        {{- end }}
        {{- if index $v "settings" "tolerations" }}
        tolerations:
          {{- toYaml (index $v "settings" "tolerations") | nindent 8 }}
        {{- end }}
```